doc/cephadm/service-management.rst

```diff
@@ -94,6 +96,10 @@ In this example, the properties of this service specification are:
     The name of the service.
 * ``placement``
     See :ref:`orchestrator-cli-placement-spec`.
+* ``networks``: A list of network identifies instructing the daemons to only bind
+    on that particular network. In case the cluster is distributed across multiple
```

Comment:
```suggestion
* ``networks``: A list of network identities instructing the daemons to only bind
    on the particular networks in that list. In case the cluster is distributed across multiple
```